.docs/angular-meteor/client/views/steps/tutorial.step_05.html

```diff
@@ -97,7 +97,7 @@
 Meteor scans all the HTML files in your directory for three top-level elements: HEAD, BODY, and TEMPLATE.
 The head and body sections are separately concatenated into a single head and body, which are transmitted to the client on initial page load.
 
-Meteor converts both the templates and body content and generates a client-side JavaScript file (bundle-time, with htmlJS) and makes the template available under the Template namespace.
+Meteor converts both the templates and body content and generates a client-side JavaScript file (bundle-time, with html & JS) and makes the template available under the Template namespace.
```

Comment:
This is the HTMLJS library.  maybe instead it's better to add a link to the package inside Meteor?